src/kubernetes/nodes_test.go

```diff
@@ -99,11 +98,11 @@ func (suite *KubeCtlNodeTestSuite) TestGetInvalidNodeExternalIP() {
 		},
 	}
 	_, err := suite.fakeclient.CoreV1().Nodes().Create(&node)
-	assert.NoError(suite.T(), err)
+	suite.NoError(err)
 
 	nodeIP, err := suite.kubectl.GetNodeExternalIP("K8S-Node-0")
-	assert.Error(suite.T(), err)
-	assert.Equal(suite.T(), "", nodeIP)
+	suite.Error(err)
+	suite.Equal("", nodeIP)
 }
 
 func (suite *KubeCtlNodeTestSuite) TearDownTest() {}
```

Comment:
Use the TearDownSuite for all test cases

Reply:
Also change the `SetupTest` to `SetupuSuite` for the whole suite not for each test cases